cristin-person/src/test/java/no/unit/nva/cristin/person/client/CristinPersonQueryTest.java

```diff
@@ -0,0 +1,38 @@
+package no.unit.nva.cristin.person.client;
+
+import static no.unit.nva.cristin.model.JsonPropertyNames.NUMBER_OF_RESULTS;
+import static no.unit.nva.cristin.model.JsonPropertyNames.PAGE;
+import static no.unit.nva.cristin.model.JsonPropertyNames.QUERY;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import java.net.URI;
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+
+public class CristinPersonQueryTest {
```

Comment:
This test is tests things that are tested elsewhere, I would remove it (and any other non-handler tests), or create a test in the handler that exposes what you're trying to test here.

Reply:
It is not easy to test in another way  because it is only used internally and not returned to the client

Reply:
Found a way to test. Deleted class